pages/index.tsx

```diff
@@ -24,40 +24,40 @@ export default function Home() {
         <div className="p-8 h-full container max-w-5xl flex flex-col justify-center">
           <div className="flex flex-col sm:flex-row items-start sm:items-end gap-1 sm:gap-6">
             <h1 className="text-5xl md:text-6xl font-extrabold">
-              BlairHacks Mini &#39;22
+              BlairHacks Mini &apos;22
             </h1>
             <a href="https://poolesvillehacks.tech" target="_blank" rel="noopener noreferrer">
-              <div className="px-2 pt-1.5 pb-0.5 rounded-lg bg-neutral-500/30">
+              <div className="px-2 pt-1.5 pb-0.5 rounded-lg bg-neutral-700/50">
                 <p className="text-sm whitespace-nowrap -mb-1.5">
                   in collaboration with
                 </p>
-                <p className="text-md md:text-xl font-medium">
+                <p className="text-md md:text-xl font-medium text-amber-300">
                   poolesville_hacks
                 </p>
               </div>
             </a>
           </div>
           <p className="text-xl md:text-2xl font-semibold mt-4 sm:mt-0.5">
-            An IRL day of coding and making for high-schoolers in MoCo
```

Comment:
My opinions on this:
- Keep "IRL" to signify that it's an IRL event esp coming out of covid, maybe not necessary?
- Keep "coding" to make it clear that it's a coding event, just "making" makes it a bit ambiguous